app/src/pages/inside/projectSettingsPageContainer/content/testLibrary/testData/datasets/dataset/dataset.jsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { defineMessages, useIntl } from 'react-intl';
+import Parser from 'html-react-parser';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import { Button } from '@reportportal/ui-kit';
+import FlagIcon from 'common/img/flag-inline.svg';
+import ArrowDownIcon from 'common/img/arrow-down-thin-inline.svg';
+import PencilIcon from 'common/img/pencil-empty-inline.svg';
+import DownloadIcon from 'common/img/download-square-inline.svg';
+import TrashcanIcon from 'common/img/trashcan-x-inline.svg';
+import plusIcon from 'common/img/plus-button-inline.svg';
+import { messages as datasetsMessages } from '../datasets';
+import styles from './dataset.scss';
+
+const cx = classNames.bind(styles);
+
+export const datasetShape = PropTypes.shape({
+  datasetName: PropTypes.string,
+  timestamp: PropTypes.number,
+  username: PropTypes.string,
+});
+
+const messages = defineMessages({
+  createdBy: {
+    id: 'testData.createdBy',
+    defaultMessage: 'Created by {username} on {date} at {time}',
+  },
+});
+
+export const Dataset = ({ dataset: { datasetName, timestamp, username } }) => {
+  const { formatMessage, formatDate, formatTime } = useIntl();
+
+  return (
+    <div className={cx('dataset__wrapper')}>
+      <div className={cx('dataset__name-tools-wrapper')}>
+        <div>
+          <div className={cx('dataset__name')}>
+            {Parser(FlagIcon)}
+            {datasetName}
+            {Parser(ArrowDownIcon)}
+          </div>
+          <div className={cx('dataset__timestamp')}>
+            {formatMessage(messages.createdBy, {
+              username,
+              date: formatDate(timestamp).replace(/\//g, '.'),
```

Comment:
Can you clarify why do you need to use `replace` here?

Reply:
to use dots as separator for date (21.02.2025) as per design because I couldn't find a way to do it through intl without overriding locale

Reply:
@AmsterGet, is this okay approach? I wonder if you have similar stuff already in the project

Reply:
Discussed in Teams chat.